src/models/JobRole.ts

```diff
@@ -0,0 +1,12 @@
+export type JobRole = {
```

Comment:
is this needed?

Reply:
Removed and using JobRoleResponse instead